README.md

```diff
@@ -1 +1,181 @@
-# Hello_Microverse_Project
\ No newline at end of file
+<a name="Hello Microverse Project"></a>
+
+
+<div align="center">
+  
+
+  <h3><b>Hello Microverse Project</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+
+# 📖 [Hello Microverse Project] <a name="about-project"></a>
+
+> A portfolio I built with html and css.
+
+
+## 🛠 Built With <a name="built-with"></a>
+1. html
+2. css
+3. Linters
+
+### Tech Stack <a name="tech-stack"></a>
+1. html
+2. css
+3. Linters
+
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://indexhtml.org/">HTML</a></li>
+    <li><a href="https://styles=css/">CSS</a></li>
+    <li><a href="https://linters/">Linters</a></li>
+  </ul>
+</details>
+
+
+### Key Features <a name="key-features"></a>
+
+
+Added index.html file
+Added style.css file
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+Live-demo will br added soon.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+Web browser
+Code editor
+
+
+
+### Setup
+
+Clone this repository to your desired folder:
+Run this command:
+git clone https://github.com/Kwamay/Hello_Microverse_Project.git
+
+
+
+### Install
+
+Install this project with:
+
+cd my-project
+npm install
+
+
+### Usage
+
+To run the project, execute the following command:
+
+ Open index.html using live server extension.
+
+### Run tests
+
+To run tests, run the following command:
+
+Coming soon
+
+### Deployment
+
+You can deploy this project using:
+
+Coming soon
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```

Comment:
Similar to the previous comment, you can feel free to remove these sections from the readme since they don't apply to this project. 👍